omegaconf/omegaconf.py

```diff
@@ -590,6 +591,19 @@ def to_container(
                (DictConfigs backed by a dataclass)
         :return: A dict or a list representing this config as a primitive container.
         """
+        if exclude_structured_configs is not None:
+            warnings.warn(
+                dedent(
+                    """\
+                The exclude_structured_configs argument to to_container is deprecated.
+                See https://github.com/omry/omegaconf/issues/548 for migration instructions.
+                """
+                ),
+                DeprecationWarning,
+                stacklevel=2,
+            )
+            if exclude_structured_configs is True:
+                structured_config_mode = SCMode.DICT_CONFIG
```

Comment:
I am leaning toward killing exclude_structured_configs before 2.1 is released.
It's a small breaking change, but right now only effects Hydra 1.1 dev release for people using partial instantiation mode.

I make the the short term upgrade smoother we can temporarily deprecate the flag but let's follow-up and remove it before 2.1 is released.

Reply:
Ok, sounds good.

Reply:
I made an issue to keep track of killing exclude_structured_configs: #556